enos/modules/test_e2e_docker/main.tf

```diff
@@ -172,6 +172,10 @@ variable "worker_tag_collocated" {
   type    = string
   default = ""
 }
+variable "max_page_size" {
```

Comment:
We will need to set a default value here since not all scenarios will pass this value into this module.

Reply:
Hm, I really wanted that to not have a default value so that it had to be set explicitly and we couldn't accidentally miss it. I'll just add it to the places that are erroring if that's ok?

Reply:
Yea that works, too.

Reply:
We'll just need to update scenarios in `boundary-enterprise`, too, when this gets merged.